File: Swap_two_numbers/Swap_two_numbers_SHORTEN.c
#include <stdio.h>

int main( void )
{
    /* INITIALIZATION */
    // this code also works for char type variables
    int a = 1;
    int b = 0;

    printf( "a: %d b: %d\n", a, b );

    /* SWAP TWO VARIABLES */
    // For detailed explanations, see 'Swap_two_numbers.c' in same folder
    a^=b^=a^=b; // a ^= (b ^= (a ^= b));

    printf( "a: %d b: %d\n", a, b );

    return 0;
}

File: Swap_two_numbers/Swap_two_numbers_SHORTEN.js
/* INITIALIZATION */
let a = 1;
let b = 0;

console.log(`a: ${a}, b: ${b}`);

/* SWAP TWO VARIABLES */
// For detailed explanations, see 'Swap_two_numbers.js' in same folder
a^=b^=a^=b; // a ^= (b ^= (a ^= b));

console.log(`a: ${a}, b: ${b}`);